src/components/MessageList/MessageListNotifications.tsx

```diff
@@ -29,8 +36,11 @@ export const MessageListNotifications = (props: MessageListNotificationsProps) =
         </CustomNotification>
       ))}
       <ConnectionStatus />
-      <MessageNotification onClick={scrollToBottom} showNotification={hasNewMessages}>
-        {t('New Messages!')}
+      <MessageNotification
+        onClick={scrollToBottom}
+        showNotification={hasNewMessages || isNotAtLatestMessageSet}
```

Comment:
I think from a UX standpoint this is not always comfortable to the user because the latest message set can be huge, with lots of messages, maybe the notification could be shown if the user scrolled up (or scrolled up x pixels) (the Angular SDK uses similar logic)

Reply:
👍🏼 I like the suggestion. Can you link the angular logic? I will copy it.

Reply:
https://github.com/GetStream/stream-chat-angular/blob/master/projects/stream-chat-angular/src/lib/message-list/message-list.component.ts#L231 - the `isUserScrolledUpThreshold` is not there as a UX feature, it's there because of the image loading - scrollbar problems

Reply:
I pulled that into a separate smaller task so that I can release that: https://github.com/GetStream/stream-chat-react/issues/1490